javascript/javascript3/week1/GiphyApi.js

```diff
@@ -0,0 +1,42 @@
+
+const search = document.getElementById('searchText')
+const resultNumber = document.getElementById('number')
+const button = document.querySelector('button')
```

Comment:
It is recommended to create a class for button and use it in js file instead of using a tag name